code/modules/vending/kinkmate.dm

```diff
@@ -40,7 +40,8 @@
 				/obj/item/storage/pill_bottle/butt_enlargement = 10,
 				/obj/item/reagent_containers/glass/bottle/crocin = 10,
 				/obj/item/reagent_containers/glass/bottle/camphor = 5,
-				/obj/item/storage/daki = 4
+				/obj/item/storage/daki = 4,
+				/obj/item/storage/box/aphrodisiac_pump = 4
```

Comment:
This can also be added modularly

Reply:
Gotta start putting modularization in my todo list